enzy_htp/mutation/api.py

```diff
@@ -367,8 +371,62 @@ def mutate_stru_with_tleap(
 
     return stru_cpy
 
+def mutate_stru_with_pymol(
+        stru: Structure,
+        mutant: List[Mutation],
+        in_place: bool = False,
+        ) -> Structure:
+    """mutate the {stru} to its {mutant} structure using PyMOL.
+    Args:
+        stru: the 'WT' structure
+        mutant: a list of Mutation() which describes a mutant to the 'WT'
+        in_place: if make the changes to the structure in-place 
+    Returns:
+        The mutated structure.
+    """
+    
+    # san check
+    for mut in mutant:
+        if not isinstance(mut, Mutation):
+            _LOGGER.error(
+                f"mutant takes only a list of Mutation(). Current mutant is: {mutant}")
+            raise TypeError
+ 
+    stru_cpy = copy.deepcopy(stru)
+
+    if_retain_order = True
+
+    # if structure is multichain, set retain_order to false and raise a flag to mark multichain structure
+    # for further treatment
+    if stru_cpy.num_chains > 2:
+        if_multichain = True
+        if_retain_order = False
+    else:
+        if_multichain = False
```

Comment:
let's just always don't use retain_order here and address the mess-up atom order in next PR.